hardhat.config.ts

```diff
@@ -73,8 +75,12 @@ const config: HardhatUserConfig = {
                 disambiguatePaths: false,
                 runOnCompile: false,
                 strict: true
+        },
+        gasReporter: {
```

Comment:
This and the `contractSizer` config are indented too far

Reply:
The indentation looks the same as the other config to me, perhaps I am missing something ? @hswopeams ? Can we accept this PR and then i can fix it in a subsequent one?

Reply:
strange that the linter didn't pick it up... could it be tabs and spaces ....

Reply:
<img width="2112" alt="Screenshot 2021-11-01 at 10 48 52" src="https://user-images.githubusercontent.com/699036/139660400-4be25cc9-5e98-4468-9971-a09609463f05.png">

Reply:
ok, so it is a tabs vs spaces thing. I have my IDE configured to represent tabs as 4 spaces and to never write out tabs, as tabs can be different on different machines. I will use tabs now, but we should fix this in the future and have our linter take care of this ...

Reply:
<img width="1392" alt="Screenshot 2021-11-01 at 11 04 30" src="https://user-images.githubusercontent.com/699036/139662272-567d169d-32d2-40fa-9834-e363f7192ccf.png">

Reply:
This should be fixed now @hswopeams